### ISSUE TITLE
The save button freezes when one tries to create a new plan RVL-1051

### ISSUE THREAD

**Issue by Naima-Bashir** [onaio/reveal-frontend#1115]
Trying to set a plan as active during creation causes the Save button to freeze with the text Saving and an error pop's up
![image](https://user-images.githubusercontent.com/61143692/89379230-158fbc00-d6fe-11ea-9277-bf10c9c1c0ba.png)

**Comment by moshthepitt** [comment]
@ciremusyoka please investigate to determine if there are any fixes that can be done on the frontend codebase.  Additionally, raise any issues on the OpenSRP server web repo.

**Comment by ciremusyoka** [comment]
@Naima-Bashir am not able to replicate this bug.
@kelvin-muchiri @p-netm can you please try if you can replicate it.

**Comment by Naima-Bashir** [comment]
Just retested this. The save button works as expected. One can now create new plans and save them.

**Comment by ciremusyoka** [comment]
@moshthepitt i will enable the save button if an error occurs so that a user can try saving again. My assumption is the error will not be from the data we are posting but from API since the save button is only enabled when all fields are filled correctly.

**Comment by moshthepitt** [comment]
@ciremusyoka not sure what you mean, could you rephrase that?

**Comment by ciremusyoka** [comment]
@moshthepitt  Currently if an error occurs when submitting a plan the save plan button remains disabled. My thinking is if an errors occures we enable the save plan button so that a user can try saving the plan again.
